lib/interface/utils.js

```diff
@@ -55,7 +56,7 @@ module.exports = {
                     }
                 } else {
                     try {
-                            callback(JSON.parse(body));
+                            callback(null, JSON.parse(body));
```

Comment:
Why did you change this to pass it in as part of the body? If it's not one of the success status codes, shouldn't it be passed back to the user as an error?